audio/out/ao_openal.c

```diff
@@ -267,6 +264,7 @@ static int init(struct ao *ao)
         goto err_out;
     }
 
+    ao->device_buffer = p->num_buffers * p->num_samples / ao->channels.num;
```

Comment:
That doesn't look right (but not sure). device_buffer is in PCM frames (what most of the player calls "samples"), which means 1 sample for each channel. E.g. `size_in_bytes = ao_device_buffer * channels.num * 2` for s16.

Reply:
You are right, according to the documentation `num_samples` are actually frames (complete samples):
https://mpv.io/manual/master/#audio-output-drivers-openal-num-samples

Reply:
Done.